docker/generated/README.md

```diff
@@ -1,5 +1,5 @@
+<!-- P/S: even this file is autogenerated :happy: -->
 ## GENERATED DIRECTORY - DO NOT EDIT
 
-The files are maintained by `manager.py`, which generates Dockerfile from `templates`.
+These files are maintained by [`manager`](../manager), which generates Dockerfile from `templates`.
```

Comment:
these links are not going to work when pushed to docker io?

Reply:
we won't push this readme to docker.io This is to warn not to remove this directory